packages/dashboard/src/components/emailPreviewHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import { SxProps, TextField, Theme, useTheme } from "@mui/material";
+import escapeHTML from "escape-html";
+import React from "react";
+
+interface EmailPreviewHeaderProps {
```

Comment:
Awesome, glad that you pulled this out! :)

Reply:
Thank you Max 😄